ridgeback_control/launch/teleop.launch

```diff
@@ -20,8 +20,11 @@
     </node>
   </group>
 
+  <arg name="config" value="planar" />
```

Comment:
Consider:

    <arg name="config" default="planar" />

Which will allow it to be overridden.